src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_COUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[BALL_COUNT];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
```suggestion
            System.out.println(balls[i]);
```

There is no need to call the object's toString() method when passing it to the print() method. If you look at the implementation of the method, you can see that toString() is called in the method.